source/docs/software/hardware-apis/sensors/encoders-software.rst

```diff
@@ -115,15 +125,15 @@ The :code:`Encoder` class offers a number of configuration methods:
         // Can be between 1 and 127 samples
         encoder.SetSamplesToAverage(5);
 
-Reading information from encoders
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Reading information from quadrature encoders
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The :code:`Encoder` class provides a wealth of information to the user about the motion of the encoder.
 
-Distance
-~~~~~~~~
+Quadrature Distance
```

Comment:
I'm not sure the adjective applies well here.

Reply:
Need unique labels

Reply:
"Reading distance from X encoder"?